CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Adds
 
 * Emit a `beforeInsert` event from the `@apostrophecms:attachment` module, with `req` and the `doc` as arguments, in order to give the possibility to override the attachment.
+* Convert uppercase URLs automatically to their lowercase version - can be disabled through `forceLowerCaseUrls` option in `@apostrophecms/page/index.js`.
```

Comment:
I think `redirectFailedUpperCaseUrls` because we don't really "force" this, we only try it if we're about to 404. And clarify that they should set that to `false` to disable the default (which is `true`).

Reply:
Done